### DISCUSSION TITLE
Table columns show up above its container when using `useWindowScroll`

### DISCUSSION THREAD

**Discussion by adrienprsn** [petyosi/react-virtuoso#653] [Q&A]
https://codesandbox.io/s/sandpack-project-forked-ejukxc

Hi !

Thank you for this great library.
I'm using `useWindowScroll` and I need to maintain some margin above the Table, like in the example.
The issue I'm facing is the rows show up above the headers.
Is there a way to prevent that ?

Cheers

**Comment by shinenic** [comment]
@adrienprsn 
Just comment out this style, lol
![image](https://user-images.githubusercontent.com/20126676/167163875-b61bfcf4-5a8f-4d34-9836-cec90a65ec23.png)

**Reply by adrienprsn** [reply]
You're missing the point, in the question I specifically say "I need to maintain some margin above the Table"

**Reply by shinenic** [reply]
oh, sorry for the misunderstanding,
I am not sure why you need the `margin`,
but I think maybe you can add a placeholder outside the table,
or add a row on the `thead` to achieve it.

for example:
https://codesandbox.io/s/sandpack-project-forked-xj4zyk?file=/App.js

**Reply by adrienprsn** [reply]
I tried that before but this doesn't work because the Table also has a box-shadow all around, like here: https://codesandbox.io/s/sandpack-project-forked-egc04d?file=/Table.js
You can see the shadow on the placeholder as you scroll down
